core/templates/domain/utilities/language-util.service.ts

```diff
@@ -145,6 +147,15 @@ export class LanguageUtilService {
     const language = this.getSupportedAudioLanguages()[audioLanguageCode];
     return language ? language.description : null;
   }
+  getLanguageDirection(languageCode: string): string {
+    let languageDirection = 'ltr';
```

Comment:
Why is it set to default, are we expecting to have directions for all the langaugeCodes, right?

Reply:
there should be directions for all language codes, but I wasn't sure what behavior would be expected for an invalid language code.
But on second thought, returning "auto" would be safer (defaults to parent element direction)